Tests/NIOTests/HappyEyeballsTest.swift

```diff
@@ -129,15 +129,15 @@ private extension SocketAddress {
         var v4addr = in_addr()
         var v6addr = in6_addr()
 
-        if inet_pton(AF_INET, ipAddress, &v4addr) == 1 {
+        if inet_pton(NIOBSDSocket.AddressFamily.inet.rawValue, ipAddress, &v4addr) == 1 {
             var sockaddr = sockaddr_in()
-            sockaddr.sin_family = sa_family_t(AF_INET)
+            sockaddr.sin_family = sa_family_t(NIOBSDSocket.AddressFamily.inet.rawValue)
             sockaddr.sin_port = in_port_t(port).bigEndian
             sockaddr.sin_addr = v4addr
             self = .init(sockaddr, host: host)
-        } else if inet_pton(AF_INET6, ipAddress, &v6addr) == 1 {
+        } else if inet_pton(NIOBSDSocket.AddressFamily.inet6.rawValue, ipAddress, &v6addr) == 1 {
```

Comment:
Similarly do you want to use the helpers in this file?

Reply:
Yeap, this should be through the wrappers.